Ensure the created revision uses the correct label

vizzv opened the pull request:
## Scope

What's changed:

- Corrected the revision sidebar label for **create** revision entries.
- Replaced the incorrect `revision_delta_updated` translation key with `revision_delta_created`.
- Ensures newly created items display **“Created”** instead of **“Updated”** in the Revisions panel across all supported databases.

## Potential Risks / Drawbacks

- Very low risk: this change only updates the translation key used at runtime and does not modify revision logic or database behavior.

## Tested Scenarios

- Created new items in **SQLite**.
- Verified the revision sidebar now correctly displays **"Created"** for creation events.

## Review Notes / Questions

- Although the translation key `revision_delta_created` currently resolves to `"Created"` and does not include a `{count}` placeholder, I kept the existing pattern:

```js
  t('revision_delta_created', revisionCount.value);
```

This preserves consistency with other revision actions and ensures future compatibility with:

- locales that may use {count}
- potential pluralization changes
- consistent function signatures across all actions

Fixes #26273

If there's anything you'd like adjusted or improved, feel free to let me know!

vizzv committed:
fix(revisions): use correct label for create revision (#26273)

vizzv committed:
Add vizzv to contributors list

vizzv committed:
Changes to be committed:
    new file:   .changeset/orange-rats-burn.md

vizzv committed:
Merge branch 'dev' of https://github.com/vizzv/directus into dev

formfcw commented on app/src/views/private/components/revision-item.vue:
```diff
@@ -30,7 +30,7 @@ const revisionCount = computed(() => {
 	switch (props.revision.activity.action.toLowerCase()) {
 		case 'create':
-			return t('revision_delta_updated', revisionCount.value);
+			return t('revision_delta_created', revisionCount.value);
```
Let’s remove the parameter here since there is none for the `revision_delta_created` message.

```suggestion
			return t('revision_delta_created');
```

formfcw commented on .changeset/orange-rats-burn.md:
```diff
@@ -0,0 +1,5 @@
+---
+'@directus/app': major
```
Let’s use "patch" since it is a fix.

```suggestion
'@directus/app': patch
```

formfcw reviewed (changes requested):
Thank you @vizzv for your first contribution :tada:

Just a few minor things.

vizzv committed:
remove unnecessary parameter from revision_delta_created

fix(revisions): remove parameter since revision_delta_created has no count placeholder

Co-authored-by: Florian C. Wachmann <dev@formfcw.com>

vizzv committed:
change changset to patch since this is a fix

change changset type to patch since this is a fix

Co-authored-by: Florian C. Wachmann <dev@formfcw.com>

vizzv committed:
Update .changeset to rewrite description in past tense

rewrite changeset description in past tense

Co-authored-by: Florian C. Wachmann <dev@formfcw.com>

vizzv commented:
Thank you! All changes have been applied.

formfcw reviewed (approved):
Thank you @vizzv :heart:

LGTM 🎉
